.github/workflows/ci.yaml

```diff
@@ -361,7 +361,7 @@ jobs:
       - name: Run isort
         run: |
           . venv/bin/activate
-          pre-commit run --hook-stage manual isort --all-files --show-diff-on-failure
+          pre-commit run --hook-stage manual isort --all-files --show-diff-on-failure --profile black
```

Comment:
The profile should not be specified here, as this is not configuration but CI.

See: <https://github.com/home-assistant/core/blob/dev/pyproject.toml#L7>

Reply:
The problem is that this CI is causing my PR to be rejected all the time because isort fails. So pre-commit checks run successfully on my commits/push but fail in the CI workflow tests 

![Captura de pantalla 2021-01-21 a las 10 18 43](https://user-images.githubusercontent.com/22393167/105330037-1d6eea00-5bd2-11eb-9b29-b53b92c68fc9.png)

Reply:
Oops, I understand what you refer to now. I hadn't noticed I was changing pre-commit invocation instead of isort

Reply:
maybe you need to rebase? As in, running an outdated version of either black or isort in some way?

Reply:
Please ping me if you have a PR open that has that issue, so I can take a look. 👍